service/gcs/runtime/runc/runc.go

```diff
@@ -324,13 +325,20 @@ func (c *container) GetAllProcesses() ([]runtime.ContainerProcessState, error) {
 	if err != nil {
 		return nil, err
 	}
+	l := logrus.WithField("cid", c.id)
+	l.WithField("pids", runningPids).Debug("running container pids")
 
 	pidMap := map[int]*runtime.ContainerProcessState{}
 	// Initialize all processes with a pid and command, leaving CreatedByRuntime
 	// and IsZombie at the default value of false.
 	for _, pid := range runningPids {
 		command, err := c.r.getProcessCommand(pid)
 		if err != nil {
+			if errors.Is(err, unix.ENOENT) {
+				// process has exited between getting the running pids above
+				// and now, ignore error
+				continue
+			}
 			return nil, err
```

Comment:
We don't want the same wrapped error here? https://github.com/microsoft/opengcs/pull/394/files#diff-e88ce53c5c78c98d276a0acc3ab891fb7c59db59a74f41262fa2ef160a84b4aaR373

Reply:
I think I'm going to remove both wraps. We don't necessarily know at that point if the process *still* exists.